### PR TITLE
Bugfix/cluster delay error

### PR DESCRIPTION
**Pull request by pablomarga**
Hi team! This PR solves the delay problem restarting the cluster when you edit a rule.

Closes [#2640](https://github.com/wazuh/wazuh-kibana-app/issues/2640)

### CODE REVIEW

File: public/components/common/restart-cluster-manager-callout.tsx [hunk 1]
```diff
@@ -65,9 +65,8 @@ class WzRestartClusterManagerCallout extends Component<IWzRestartClusterManagerC
     try{
       this.setState({ warningRestarting: true, warningRestartModalVisible: false});
       const data = await restartClusterOrManager(this.props.updateWazuhNotReadyYet);
-      // this.setState({ warningRestarting: false });
       this.props.onRestarted();
-      this.showToast('success', `${data.restarted === 'cluster' ? 'Restarting cluster, it will take up to 30 seconds.': 'Manager was restarted'}`)
+      this.showToast('success', `${data.restarted === 'cluster' ? 'Cluster was restarted': 'Manager was restarted'}`)
```
Comment (jsanchez91) [inline public/components/common/restart-cluster-manager-callout.tsx:69]: If the possible values of `data.restarted` are `cluster` or `manager` change the string template to:

```ts
`${data.restarted} was restarted`
```

**Review by jsanchez91 — CHANGES_REQUESTED** [review]
Please check the requested changes and update the CHANGELOG file with the Issue.

Remember to update your branch with the last changes in the branch `4.0-7.9`

File: server/controllers/wazuh-api.js [hunk 2]
```diff
@@ -975,16 +975,13 @@ export class WazuhApiCtrl {
...
-          type: 'request',
-          method,
-          fullUrl,
-          data,
-          options,
-        });
+        const del = data.delay;
+        delete data.delay;
+        setTimeout(() => {
+          this.makeRequest(method, path, data, id, reply, token).then(console.log).catch(console.log);
```
Comment (Desvelao) [inline server/controllers/wazuh-api.js:983]: I think we should log here, maybe.

**Review by jsanchez91 — APPROVED** [review]
LGTM!

**Review by Desvelao — APPROVED** [review]
LGTM!

**Pull request merged**